tests/test-tools.sh

```diff
@@ -358,11 +523,13 @@ function test-tools-all-pro() {
 }
 
 # Print a simple help message, describing the options for this script
-function test-tools-help() {
-    echo -e "\nUsage: ./test-tools.sh test-tools-{build|build-pro|init|debug|server|all|shutdown|all|help}\n"
-    echo -e "This script is mainly intended to provide useful functions that may be called"
-    echo -e "  by a variety of other test scripts, e.g., <voltdb>/tests/sqlgrammar/run.sh,"
-    echo -e "  but it may also be called directly on the command line."
+function tt-help() {
+    echo -e "\nUsage: ./test-tools.sh test-tools-{build[-pro]|init|debug|server[-pro]|all[-pro]|shutdown}"
```

Comment:
For such chunk of message to print, I would prefer doing it like:
```suggestion
    echo -e <<EOF
    
Usage: ./test-tools.sh test-tools-{build[-pro]|init|debug...
...
but options usually call other options that are prerequisites.

EOF
```